openfisca_us/variables/hhs/hhs_smi.py

```diff
@@ -0,0 +1,36 @@
+from openfisca_core.model_api import *
+
+
+class hhs_smi(Variable):
+    value_type = float
+    entity = SPMUnit
+    label = u"HHS SMI"
```

Comment:
```suggestion
    label = u"State median income as defined by the Department of Health and Human Services"
    documentation = u"SPM unit's median income as defined by the Department of Health and Human Services, based on their state and size"
```

Reply:
I'd go for a slightly shorter "State median income (DHHS)" label